cypress/integration/pages/testsForAllPages.js

```diff
@@ -365,6 +365,20 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
           .should('have.attr', 'href', '#content');
       });
 
+      if (variant === 'default') {
```

Comment:
This is to test the script link does not render for non variant services

Reply:
The `it('...', ...)` explains why this is the case now.